tools/webdriver/webdriver/transport.py

```diff
@@ -27,17 +29,12 @@ def from_http_response(cls, http_response):
         # >       "application/json; charset=utf-8"
         # >    "cache-control"
         # >       "no-cache"
-        assert http_response.getheader("Content-Type") == "application/json; charset=utf-8"
-        assert http_response.getheader("Cache-Control") == "no-cache"
```

Comment:
After discussing on IRC with the developer of the chromedriver, we agreed to ignore the headers here, provided there's a test for them elsewhere.